### ISSUE TITLE
[bug]Map pins are extremely large

### ISSUE THREAD

**Issue by shakedlv** [ONEARMY/community-platform#2708]
When viewing as Project kamp theme some map pin are extremely large 

Plastic and Fashion map view:
![1](https://github.com/ONEARMY/community-platform/assets/88069759/ade5ae64-4f4f-46d1-b302-d2bd975f7c62)
![2](https://github.com/ONEARMY/community-platform/assets/88069759/f44bca51-4d4c-47aa-a869-9d76833681a4)
Project Kemp map view: 
![3](https://github.com/ONEARMY/community-platform/assets/88069759/3923b337-e1ee-4be7-9961-5cb4a46e79b3)

**Comment by shakedlv** [comment]
Could not find the cause for this bug.
temporary fix;
file : src/pages/Maps/Content/View/Sprite.tsx
```
  /**
   * Fixed Bug #2708 : When theme was Project Kamp the pin was extremly large
   * coulndt find reason for this right now.
   * force a style directly for the img tag itself.
   */
  return L.divIcon({
    className: `icon-marker icon-${pin.type}`,
    html: `<img src="${icon}" style="max-width:37px!important; max-height:37px!important;"/>`,
    iconSize: L.point(38, 38, true),
  })
}
```

**Comment by davehakkens** [comment]
Thanks for flagging. But not really an issue at the moment since the Project Kamp instance doesnt use the map. We will revist this when we will use it!
